protobuf/compact_activity_snapshot.proto

```diff
@@ -363,3 +364,12 @@ message VacuumProgressStatistic {
   int64 indexes_total = 11; // Postgres 17+
   int64 indexes_processed = 12; // Postgres 17+
 }
+
+message QueryRun {
```

Comment:
I didn't see this well previously when Maciek mentioned, but now I'm reviewing the code, it is indeed confusing to see 3 things are named the same and representing slightly different things:
* QueryRun in server_message.proto: websocket incoming message
* QueryRun in compact_activity_snapshot.prot: websocket outgoing message
* QueryRun in state.go: kinda mix of both

That said, I don't think having three (or two) different names will help here that much honestly, or I at least don't have better suggestions here, so I think it's fine to keep it as is.